src/components/FileUpload/Card.js

```diff
@@ -0,0 +1,24 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import styled, { css } from 'styled-components';
+
+const StyledCard = styled.div`
+  width: 100%;
+
+  ${(props) =>
+    props.state === 'error' &&
+    css`
+      border-color: #ff3b3b;
+    `};
+`;
+
+function Card(props) {
```

Comment:
this does nothing, you can just name the StyledCard constant `Card` and set the displayName (for ReactDevTools) manually:
`Card.displayName = 'Card';`